packages/browserify/test/util.js

```diff
@@ -174,17 +156,16 @@ async function prepareBrowserifyScenarioOnDisk ({ scenario }) {
     )
     runBrowserifyPath = `${__dirname}/fixtures/runBrowserifyBundleFactor.js`
   }
-  // yarn link workspace dependencies
+
+  // install must happen before link, otherwise npm will remove any linked packages upon install
+  const installDevDepsResult = spawnSync('npm', ['install', '--no-save', ...depsToInstall], { cwd: projectDir })
   overrideDepsWithLocalPackages(projectDir)
-  // limit concurrency so that yarn v1 doesnt break its own cache
-  const installDevDepsResult = await limitConcurrency(async function () {
```

Comment:
unless we're running multiple processes against this code, the concurrency-limit thing _shouldn't_ be needed because a) `spawnSync` is synchronous and b) afaik `npm` does not have this problem.